src/DatePicker/DatePicker.js

```diff
@@ -158,6 +173,12 @@ class DatePicker extends Component {
     });
   }
 
+  componentDidMount() {
+    const node = ReactDOM.findDOMNode(this.refs.root);
```

Comment:
Why accessing the DOM element when we can use the synthetic React events system?